.clang-format

```diff
@@ -0,0 +1,109 @@
+# This configuration file can be used to auto-format the code base
+# To apply the coding style you can run the following command (assuming you installed clang-format-6.0 on your system):
+# $ find ./src -iregex '.*\.\(cpp\|hpp\|c\|h\|cc\|hh\|cxx\|hxx\)$' | xargs clang-format-6.0 -i -style=file
+# You can find more information on the different config parameters in this file here:
+# https://releases.llvm.org/6.0.0/tools/clang/docs/ClangFormatStyleOptions.html
+---
+TabWidth: 2
```

Comment:
still not getting why "2" here? our basic indentation is "8", hence i'd assume that our tab width would probably be best set to "8" too, even though we don't use tabs. That's also what .dir-locals.el suggests...

Reply:
The value is unused (since tabs are disabled).

Reply:
i can change thatto 8

Reply:
well, I think either it should be 8, or the line should be removed. I mean, if its not used in the best case it will just confuse the next one reading this...

Reply:
resolved here: https://github.com/systemd/systemd/pull/11690/commits/3d13edcbee52eead5f0e3f218b8511f58a3a31ac
changed to 8 characters